cove_bods/views.py

```diff
@@ -95,6 +95,18 @@ def explore_bods(request, pk):
         total_interest_statements += count
     context['statistics']['count_ownership_or_control_interest_statement'] = total_interest_statements
 
+    # The use of r_e_type is to stop flake8 complaining about line length
+    r_e_type = 'registeredEntity'
+    context['statistics']['count_entities_registeredEntity_legalEntity_with_any_identifier'] = \
+        context['statistics']['count_entity_statements_types_with_any_identifier'][r_e_type] + \
+        context['statistics']['count_entity_statements_types_with_any_identifier']['legalEntity']
+    context['statistics']['count_entities_registeredEntity_legalEntity_with_any_identifier_with_id_and_scheme'] = \
+        context['statistics']['count_entity_statements_types_with_any_identifier_with_id_and_scheme'][r_e_type] + \
+        context['statistics']['count_entity_statements_types_with_any_identifier_with_id_and_scheme']['legalEntity']
+    context['statistics']['count_entities_registeredEntity_legalEntity'] = \
+        context['statistics']['count_entity_statements_types'][r_e_type] + \
+        context['statistics']['count_entity_statements_types']['legalEntity']
```

Comment:
It's possible to do these line continuations with brackets rather than slashes.

```python
    context['statistics']['count_entities_registeredEntity_legalEntity'] = (
        context['statistics']['count_entity_statements_types'][r_e_type] +
        context['statistics']['count_entity_statements_types']['legalEntity'])
```

Reply:
Just checked, this is kinda suggested in Pip8 tho it's not totally explicit. https://www.python.org/dev/peps/pep-0008/#indentation Can we make sure this is documented in https://opendataservices.gitbook.io/developer-docs/code/code-python#coding-style so all us developers can be on the same page about this?